packages/fxa-payments-server/src/routes/Checkout/index.stories.tsx

```diff
@@ -77,4 +49,28 @@ const MOCK_PROPS: CheckoutProps = {
   fetchCheckoutRouteResources: action('fetchCheckoutRouteResources'),
 };
 
-init();
+const storyWithProps = (routeProps: CheckoutProps) => {
+  const story = () => (
+    <CheckoutRoute routeProps={routeProps} />
+  )
+  return story;
+};
+
+export const SubscriptingWithANewAccount = storyWithProps({...MOCK_PROPS});
```

Comment:
`SubscribingWithANewAccount` (or even `SubscribeWithNewAccount`)?
Subscripting is a word, but it feels like a weird word; although this is just a \*.stories.\* file, so&hellip; 🤷

Reply:
😅 Fingers are too used to typing "subscription", I'll fix this. Thanks!